lib/queue.go

```diff
@@ -82,7 +83,7 @@ func NewRequestQueue(processor func(ctx context.Context, item *QueueItem) (*http
 	identifier := "NoAuth"
 	if user != nil {
 		queueType = Bot
-		identifier = user.Username + "#" + user.Discrim
+		identifier = user.Id
```

Comment:
The problem with this change is that it affects metrics. It was originally the bot Id and was changed later on to User#Discrim due to it being almost impossible to search for a bot in a graph. Can you provide more context as to why this change was made?

Reply:
I forgot, will revert